### ISSUE TITLE
Wasp db studio not working?

### ISSUE THREAD

**Issue by Martinsos** [wasp-lang/wasp#228]
When I run `wasp db studio` with the latest version of Wasp, I get error from Prisma Studio (check below).

This is on Archlinux.

```
Message: Error in Prisma Client request: 

Query engine binary for current platform "debian-openssl-1.1.x" could not be found.
This probably happens, because you built Prisma Client on a different platform.
(Prisma Client looked in "/home/martin/git/wasp-lang/wasp/examples/thoughts/.wasp/out/server/node_modules/prisma/node_modules/@prisma/engines/query-engine-debian-openssl-1.1.x")

Searched Locations:




To solve this problem, add the platform "undefined" to the "generator" block in the "schema.prisma" file:
generator studio-client {
  provider      = "prisma-client-js"
  binaryTargets = ["native"]
}

Then run "prisma generate" for your changes to take effect.
Read more about deploying Prisma Client: https://pris.ly/d/client-generator
  
  Query:
  prisma.thought.findMany({
  take: 100,
  skip: 0,
  select: {
    id: true,
    textMarkdown: true,
    createdAt: true,
    updatedAt: true,
    tags: true,
  }
})
```

**Comment by Martinsos** [comment]
I am just trying this locally, and here are some conclusions:
- Prisma generator downloads binaries (engines) specific for your machine, and then many prisma commands use them. These binaries are prebuilt for different platforms, and by default Prisma figures out on its own on which platform are you and downloads the correct binary (in my case for debian although I am on archlinux but it seems to work). You can specify exact binary manually in the PSL schema if you want, but we are letting it pick the binary on its own.
- In the error above, Prisma says it is looking under `.wasp/out/server/node_modules/prisma/node_modules/@prisma/engines/query-engine-debian-openssl-1.1.x`. I checked and while `node_modules/prisma` exists in `.wasp/out/server/`, there is no further `node_modules` in it - which explains why Prisma Studio couldn't find an engine binary. On the other hand, I do see correct engine binary directly in the `node_modules/prisma`! Next, we are using `output = "../server/node_modules/.prisma/client"` in schema.prisma, under `generator`, and when I took look into that location, I could see engine binary also being present there.

So, conclusion is that engine binary is present on two locations: one which looks fairly standard and another which was specific by us via `generator.output`. However, Prisma Studio is looking under a location that is none of these two!

Question is, why is Prisma Studio not looking under location specified by `output`?

**Comment by Martinsos** [comment]
It turns out this was bug in Prisma, they fixed it in new version.

Fixed with 8d3e3830f41d638f931aeeb9e4dbbacf9c40c8b9 (I bumped the Prisma version).
